apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/ir/BackendIr.kt

```diff
@@ -120,19 +120,19 @@ internal data class BackendIr(
   }
 
   data class NamedFragment(
-      val name: String,
       val source: String,
       val comment: String,
       val selectionSet: SelectionSet,
-      val defaultImplementationSelectionKey: SelectionKey,
-      val defaultImplementationSelectionSet: SelectionSet,
+      val implementationSelectionSet: SelectionSet,
       val variables: List<Variable>,
-      ) {
+  ) {
     data class SelectionSet(
+        val name: String,
         val fields: List<Field>,
         val fragments: Fragments,
         val typeCondition: IntrospectionSchema.TypeRef,
         val possibleTypes: Set<IntrospectionSchema.TypeRef>,
+        val defaultSelectionKey: SelectionKey,
         val selectionKeys: Set<SelectionKey>,
```

Comment:
Does `selectionKeys` need to be a `Set`? I only see usages of it containing one value. It actually looks like it's always equal to `defaultSelectionKey` ? Or do I miss something?